weasyl/test/db_utils.py

```diff
@@ -33,6 +35,18 @@ def create_user(full_name="", birthday=arrow.get(586162800), config=None, userna
     add_entity(users.Profile(userid=user.userid, username=username,
                              full_name=full_name, unixtime=arrow.get(0), config=config))
     add_entity(users.UserInfo(userid=user.userid, birthday=birthday))
+    # Set a password for this user
+    if password is not None:
+        d.engine.execute("INSERT INTO authbcrypt VALUES (%(id)s, %(bcrypthash)s)",
+                         id=user.userid, bcrypthash=login.passhash(password))
+    # Set an email address for this user
+    if email_addr is not None:
+        d.engine.execute("UPDATE login SET email = %(email)s WHERE userid = %(id)s",
+                         email=email_addr, id=user.userid)
+    # Force the userID to a certain value (needed in rare circumstances, e.g., tests)
```

Comment:
“e.g. tests” is not super descriptive in a set of database utilities designed for tests, so maybe just remove this parenthesized bit. =)